Menus/MainMenu/Scripts/MainMenuFace.cs

```diff
@@ -122,48 +129,6 @@ private IEnumerator AnimateVisibility(bool show)
 
 			m_VisibilityCoroutine = null;
 		}
-
-		void BeginVisuals()
```

Comment:
I did not get a chance to discuss with @dunity, no. I saw that one of the two methods was completely unused, and found that removing the call to EndVisuals did not cause any noiticeable change.  They seem to have been replaced by AnimateVisibility. They were likely an oversight or the result of an automatic merge.